pycontrails/models/cocip/cocip_params.py

```diff
@@ -61,11 +61,12 @@ class CocipParams(ModelParams):
     #: Set to ``False`` when input Flight includes emissions data.
     process_emissions: bool = True
 
-    #: Integrate initial_contrails with time steps of dt
+    #: Apply Euler's method with a fixed step size of ``dt_integration``. Advected waypoints
+    #: are interpolated against met data once each ``dt_integration``.
     dt_integration: np.timedelta64 = np.timedelta64(30, "m")
 
-    #: Difference in altitude between top and bottom layer for stratification calculations (m)
-    #: Used to approximate derivative of "lagrangian_tendency_of_air_pressure" layer
+    #: Difference in altitude between top and bottom layer for stratification calculations,
+    #: [:math:`m`]. Used to approximate derivative of "lagrangian_tendency_of_air_pressure" layer.
     dz_m: float = 200.0
```

Comment:
I wish this parameter was named more clearly too, but I don't need to over do it. 

Does the 2012 paper use a specific variable? At least we could make sure we're consistent with the paper.